src/main.lua

```diff
@@ -553,7 +569,7 @@ end, function(msg)
 		addSupplyData(msg.ioEvent)
 	end
 
-	return prunedStateResult
+	-- return prunedStateResult -- TODO: need to return?
```

Comment:
appears tests work fine without returning here in prune. its the only function returning so the lua type annotations was calling it out -- will remove